.github/workflows/concrete_python_test_macos.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     paths:
       - .github/workflows/concrete_python_tests_macos.yml
```

Comment:
We don't want to run macos CI on each PR to release pressure on the macos runner. Moreover concrete python tests should not break, maybe we could add for compiler stuff to check only the compiler build. wydt?

Reply:
Got it!

Reply:
that's on this PR https://github.com/zama-ai/concrete/pull/1140